### ISSUE TITLE
Rename tracker to "reporter" (or similar)

### ISSUE THREAD

**Issue by benvinegar** [benvinegar/counterscale#136]
I borrowed this terminology from Fathom (`tracker.js` is a fork of that project's script), but I don't think it's good name:

1. "tracker" sounds anti-privacy
2. The script doesn't actually track you – I deleted all the cookie-related code from the script

I think `reporter.js` would be more accurate.

**Comment by stoic-hugo** [comment]
Reporter.js sounds too generic. `counter.js` based on project name sounds nice, but might be limited in scope.

**Comment by benvinegar** [comment]
Yeah `counter.js` is a good idea. Fits the name, could even work as a branding element (e.g. people see `counter.js` in the network tab, more obvious what it is).

Re: limited in scope – that's also fair. But I don't think the name has to limit what it does. For example, there could be a future where it collects basic latency/web vitals, and yeah that's not "counting" but IMO it's still fair game.

Will sit on this for a bit.

**Comment by stoic-hugo** [comment]
> Yeah `counter.js` is a good idea. Fits the name, could even work as a branding element (e.g. people see `counter.js` in the network tab, more obvious what it is).

I agree. I also thought about `count.js`, `insights.js`, `metrics.js` `cs.js` (CounterScale) etc. But keeping a part of the brandname like `counter.js` seems better.

**Comment by joshdales** [comment]
I would also vote for `counter.js`

**Comment by joshdales** [comment]
Sounds very reasonable to me
